yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -330,9 +373,6 @@ private object YarnClusterDriver extends Logging with Matchers {
 }
 
 private object YarnClasspathTest extends Logging {
-
-  var exitCode = 0
```

Comment:
Looks like this is really not needed, but in general, try to refrain from changing things unrelated to the patch (or call them out explicitly in the PR description).